flowstate_ros_bridge/include/flowstate_ros_bridge/executive.hpp

```diff
@@ -96,6 +96,7 @@ class Executive : public std::enable_shared_from_this<Executive> {
   // Get a list of behavior trees in the solution.
   absl::StatusOr<std::vector<BehaviorTree>> behavior_trees() const;
 
+  using ProcessFeedbackCallback = std::function<void(const absl::Status &)>;
```

Comment:
```suggestion
  using ProcessFeedbackCallback = std::function<void(bool done, const absl::Status &)>;
```

Also pass `done` from the current_operation.